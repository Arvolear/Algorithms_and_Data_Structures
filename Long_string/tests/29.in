5538992014203114481328607685081554544
47480991341496276235765130596