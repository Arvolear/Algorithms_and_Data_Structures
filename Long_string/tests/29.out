5538992061684105822824883920846685140
5538991966722123139832331449316423948
262996831866995097809937168112077554850382731530980091976857228224
116657042